src/cloudai/workloads/ai_dynamo/ai_dynamo.py

```
@@ -234,14 +242,19 @@ def validate_connector(cls, v: str | list[str] | None) -> str | list[str] | None
     @model_validator(mode="after")
     def populate_prefill_decode_args(self) -> "AIDynamoArgs":
         """Populate prefill/decode args."""
+        if self.mode == "aggregate" and self.backend.lower() != "vllm":
+            raise ValueError("AI Dynamo aggregate mode is currently supported only for the vLLM backend")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)